src/FormBuilder/resources/js/form-builder/src/settings/group-email-settings/index.tsx

```diff
@@ -0,0 +1,28 @@
+import {getFormBuilderWindowData} from '@givewp/form-builder/common/getWindowData';
+import EmailTemplateOptions from './email/template-options';
+import {useFormState} from '@givewp/form-builder/stores/form-state';
+
+const getEmailSettings = () => {
+    const {emailNotifications} = getFormBuilderWindowData();
+
+    return emailNotifications.map((emailNotification) => {
+        const {id, title} = emailNotification;
+
+        return {
+            name: title,
+            path: `email-settings/${id}`,
+            element: <EmailTemplateOptions notification={id} />,
+            showWhen: areEmailSettingsEnabled,
+        };
+    });
+};
+
+const areEmailSettingsEnabled = () => {
+    const {
+        settings: {emailOptionsStatus},
+    } = useFormState();
+
+    return emailOptionsStatus === 'enabled';
+};
```

Comment:
Couple things here:

1. is `showWhen` a routing feature?
2. Using a hook here would only make sense if we were inside a custom hook or component.  Since this is a function that returns an object it's best to elevate this logic.

Reply:
`showWhen` is a param a route can have containing a function that runs every time the Menu is re-rendered to determine if that route must be visible or not.

Elevated the hook to where the callback function is called: 230d25bd2d461c33d2dfd3a58d1c60ef3c4d5fc5